tsconfig.json

```diff
@@ -9,8 +9,7 @@
     "moduleResolution": "node",
     "esModuleInterop": true,
 
-    "target": "es2018",
```

Comment:
This value was being overridden by the configuration in `rollup.config.js`

Reply:
I think that we should use the tsconfig as the source of truth, rather than the rollup config, because we're more likely to have ts tooling that doesn't use rollup than we are the other way around.